Decimus/Lib/libquicr/QPublishTrackHandlerObjC.mm

```diff
@@ -20,8 +20,12 @@ -(id) initWithFullTrackName: (id<QFullTrackName>) full_track_name trackMode: (QT
     assert(handlerPtr);
     const auto ftn = handlerPtr->GetFullTrackName();
     const auto converted = [[QFullTrackNameImpl alloc] init];
-    NSData* nameSpace = [[NSData alloc] initWithBytes:(void*)ftn.name_space.data()  length:ftn.name_space.size()];
-    converted.nameSpace = nameSpace;
+    NSMutableArray<NSData*>* tuple = [NSMutableArray arrayWithCapacity:ftn.name_space.size()];
+    for (const auto& element : ftn.name_space.GetEntries()) {
+        NSData* objcElement = [[NSData alloc] initWithBytes:(void*)element.data() length:ftn.name_space.size()];
+        [tuple addObject:objcElement];
+    }
+    converted.nameSpace = tuple;
```

Comment:
`quicr::TrackNamespace::size()` returns the size in bytes of the entire namespace (since in c++ it is stored contiguously in 1 vector). I'm guessing you want `ftn.name_space.GetEntries().size()` instead, otherwise you've allocated (likely) more than you actually use.

Reply:
Nice spot, this is suppossed to be using the convinience function from above anyway.